src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/ProjectUpgradesPage/ProjectUpgradesPage.tsx

```diff
@@ -2,18 +2,24 @@ import { Trans } from '@lingui/macro'
 import { Button, Space, Statistic } from 'antd'
```

Comment:
This page is 
<img width="839" alt="Screenshot 2023-02-14 at 2 11 18 PM" src="https://user-images.githubusercontent.com/12551741/218633508-97c076b6-1fbb-4a5d-afd0-727c8e1b028f.png">

Reply:
Btw, colors are shitty, planning to revisit.